Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs

```diff
@@ -57,7 +65,11 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
 
                 factory.Configuration.Variables["configDir"] = environment.ContentRootPath;
 
-                loggingBuilder.AddNLogWeb(factory, new NLogAspNetCoreOptions { ReplaceLoggerFactory = true });
+                loggingBuilder.AddNLogWeb(factory, new NLogAspNetCoreOptions
+                {
+                    ReplaceLoggerFactory = true,
+                    RemoveLoggerFactoryFilter = false,
```

Comment:
When already using `ReplaceLoggerFactory = true` (Replacing Microsoft LoggerFactory with NLog LoggerFactory), then `RemoveLoggerFactoryFilter = false` has not effect, because it is only relevant when using Microsoft LoggerFactory.

Reply:
Thank you for the advice!